pkg/windows/windows.go

```diff
@@ -14,7 +14,7 @@ import (
 var (
 	backslashReplaceRgx    = regexp.MustCompile(`[\\/]+`)
 	windowsDriveRgx        = regexp.MustCompile("^[a-z]:/")
-	windowsNetworkMountRgx = regexp.MustCompile(`(?i)^\\\\[a-z]+`)
+	windowsNetworkMountRgx = regexp.MustCompile(`(?i)^\\\\([a-z]|((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))+`)
```

Comment:
Can this be simplified to `(?i)^\\\\((\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})|(\w+))\\`?

Reply:
How about ipv6? https://stackoverflow.com/questions/53497/regular-expression-that-matches-valid-ipv6-addresses

Reply:
`((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)` is the offical regex for ip address so that’s why I used it.

it accepts:
```
127.0.0.1
192.168.1.1
192.168.1.255
255.255.255.255
0.0.0.0
```
and rejects:
```
30.168.1.255.1
127.1
192.168.1.256
-1.2.3.4
1.1.1.1.
3...3
```

Reply:
Would you like to parse both v4 and v6?